Ginger/Ginger/UserControlsLib/UCListView/UcListView.xaml.cs

```diff
@@ -655,16 +706,16 @@ private async void xSearchClearBtn_Click(object sender, RoutedEventArgs e)
             {
                 await CancelSearchAsync();
             }
-            //else
-            //{
-            //    //if search is already complete and user trying to clear text we collapse the unselected nodes
-            //    List<TreeViewItem> pathNodes = new List<TreeViewItem>();
-            //    if (xTreeViewTree.LastSelectedTVI != null)
-            //    {
-            //        pathNodes = UCTreeView.getSelecetdItemPathNodes(xTreeViewTree.LastSelectedTVI);
-            //    }
-            //    UCTreeView.CollapseUnselectedTreeNodes(xTreeViewTree.TreeItemsCollection, pathNodes);
-            //}
+            else
+            {
+                //if search is already complete and user trying to clear text we collapse the unselected nodes
+                //List<TreeViewItem> pathNodes = new List<TreeViewItem>();
+                //if (xTreeViewTree.LastSelectedTVI != null)
+                //{
+                //    pathNodes = UCTreeView.getSelecetdItemPathNodes(xTreeViewTree.LastSelectedTVI);
+                //}
+                //UCTreeView.CollapseUnselectedTreeNodes(xTreeViewTree.TreeItemsCollection, pathNodes);
+            }
```

Comment:
please delete all commented out code + all search functions/fields which i copied from ucTreeView and not been used (like SearchAsync())